src/App.tsx

```diff
@@ -2,24 +1,49 @@
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
-
+import { Pagination } from './components/Pagination';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+
+  const HandlePageChange = (newPage: number) => {
+    if (newPage !== currentPage) {
+      setCurrentPage(newPage);
+    }
+  };
+
+  const HandlePerPageChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
Consider renaming `HandlePerPageChange` to `handlePerPageChange` to follow JavaScript naming conventions for functions.